src/Microsoft.AspNet.Mvc.Testing/ControllerTestHelperContext.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Core.Collections;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+
+namespace Microsoft.AspNet.Mvc.Testing
+{
+	public class ControllerTestHelperContext
+	{
+
+		// Not sure if this RouteData property will be usefull..
+		public RouteData RouteData
+		{
+			get;
+			set;
+		}
+
+		// Not sure if this HttpContext property will be usefull..
+		public HttpContext HttpContext
+		{
+			get;
+			set;
+		}
```

Comment:
These are both absolutely useful. My suggestion for all of these is to pre-intialize them so the user can just set values directly.